src/components/FormItem.vue

```diff
@@ -258,9 +258,10 @@ export default {
                 if (typeof rawValidator.validator === 'function') {
                     const { validator } = rawValidator;
                     const { message = 'Custom error message' } = rawValidator;
+                    const translatedMessage = this.translate(message);
 
                     if (!validator(this.localValue)) {
-                        this.errors.push(message);
+                        this.errors.push(translatedMessage);
```

Comment:
nestacilo `this.errors.push(this.translate(message));`?

Reply:
Done